.travis.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     - env: MODE="test" NUMPY="numpy=1.14"
     - env: MODE="test" NUMPY="numpy=1.15"
     - env: MODE="test" PYTHON="3.4"
-    - env: MODE="test" PYTHON="3.5"
+    - env: MODE="test" COVERAGE="true" PYTHON="3.5"
```

Comment:
I might add this build to the "Basic" stage, just so that we have both coverage builds happening there.  Otherwise I can anticipate some confusion if one of these builds runs but the other doesn't, and all of a sudden it looks like the coverage has changed if you just glance at the status checks.

Reply:
If the static checks fail but the first basic codecov build passes, would codecov still report its results? I had thought if any part of the job failed (even non-codecov related builds) then codecov would auto-fail.

Reply:
No we have it set up so that codecov always reports its status checks, regardless of whether the build passed or failed.

Reply:
Done